### ISSUE TITLE
The Dockerfile for hadolint does shows lint error on https://hadolint.github.io/hadolint/ 

### ISSUE THREAD

**Issue by anil181** [hadolint/hadolint#827]
- [x] This is a bug report
- [ ] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior
No Error should be found on linting the Dockerfile

### Actual behavior
Linting Error shown with the Dockerfile

### Steps to reproduce the behavior
1) Take the content of the Dockerfile from https://hadolint.github.io/hadolint/
2) LInt the same on https://hadolint.github.io/hadolint/
<img width="1245" alt="Screenshot 2022-04-27 at 17 07 37" src="https://user-images.githubusercontent.com/7187168/165510043-abd0b7bc-5e90-4254-8549-73d9c0ed0bf0.png">

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version` or
  `docker run --rm ghcr.io/hadolint/hadolint hadolint --version`:**

```bash
(paste your output here)
```

**Dockerfile** (if relevant)

```Dockerfile
(paste content of your Dockerfile)
```

**Additional environment details (OS, stack version, etc.)**
